website/docs/cdktf/create-and-deploy/terraform-cloud.mdx

```diff
@@ -47,6 +47,7 @@ For new projects, do the following:
 1. Choose whether to send crash reports to the CDKTF team.
 
 CDKTF creates both a scaffolded project in your chosen language and a new Terraform Cloud workspace. Your project is connected to the workspace and you can begin using CDKTF and Terraform Cloud together.
+By default, the workspace is set to Local Execution mode, which means Terraform continues to run on your local machine. If you want Terraform to run remotely, you need to set the workspace to Remote Execution mode.
```

Comment:
You could also link to the Terraform Cloud docs, something along the lines of: "To learn more about remote operations, visit the [Terraform Cloud docs](https://www.terraform.io/cloud-docs/run/remote-operations)."

But I'm sure that @laurapacilio could phrase this way better 😁